includes/wikia/services/WikiService.class.php

```diff
@@ -91,8 +91,23 @@ function() use ( $wiki, $useMaster, $excludeBots, $limit ) {
 		);
 
 		$userIds = array_unique( array_merge( $userIds, $adminIds ) );
+		$result = [];
 
-		return $userIds;
+		foreach ($userIds as $admin) {
+			$user = \User::newFromId( $admin['userId'] );
```

Comment:
As far as I can tell, `getUserIdsFromDB` returns an array of just user IDs and I don't see `$userIds` being converted into an associative array anywhere, so the key `userId` wouldn't exist and this should just be `\User::newFromId( $admin );`.

Reply:
You are right... I'm not sure why my tests on sandbox didn't reveal this issue. Probably due to the cache.

Fix in 7a94dcf184a